README.md

```diff
@@ -105,6 +105,14 @@ source /FULL/PATH/TO/wp-completion.bash
 
 Don't forget to run `source ~/.bash_profile` afterwards.
 
+If using zsh for your shell, you may need to do the following:
```

Comment:
Yes (I added to my `.zshrc` but that may be wrong) – does that not agree with my instructions here?

Without the `bashcompinit` lines, I get a command not found error for `complete` (last line of `wp-completion.bash`).